File: src/main/java/superstitio/powers/patchAndInterface/interfaces/orgasm/OnOrgasm_onCheckOrgasm.java
package superstitio.powers.patchAndInterface.interfaces.orgasm;

import superstitio.powers.SexualHeat;

public interface OnOrgasm_onCheckOrgasm extends OnOrgasm {
    /**
     * 检测高潮时的钩子
     */
    @Override
    void onCheckOrgasm(SexualHeat SexualHeatPower);
}

File: superstitioMain/java/superstitio/powers/patchAndInterface/interfaces/orgasm/OnOrgasm_onCheckOrgasm.kt
package superstitio.powers.patchAndInterface.interfaces.orgasm

import superstitio.powers.SexualHeat

interface OnOrgasm_onCheckOrgasm : OnOrgasm
{
    /**
     * 检测高潮时的钩子
     */
    override fun onCheckOrgasm(SexualHeatPower: SexualHeat)
}
